scripts/compare_phenopackets.py

```diff
@@ -0,0 +1,137 @@
+"""
+compare_phenopackets.py
+
+Provides `compare_jsons`, which compares two JSON-like structures
+(e.g., generated vs. ground-truth phenopackets) and computes:
+   - TP: True Positives (correctly extracted fields)
+   - FP: False Positives (hallucinated or extra fields)
+   - FN: False Negatives (missed or wrong fields)
+
+We ignore True Negatives because in free-form information extraction
+the "not-extracted" space is unbounded.
+
+Dependencies:
+   pip install jsoncomparison
+
+Usage:
+   from compare_phenopackets import compare_jsons
+   metrics = compare_jsons(true_json_dict, generated_json_dict_or_str)
+   print(metrics)  # -> {"TP": ..., "FP": ..., "FN": ...}
+"""
```

Comment:
We have different styles of documenting. Can we agree on one?

I would suggest leaving the heads out and using docstings in numpy style format.